From fca50cfad12edf0f5dcb752dca589783c0ab7508 Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Thu, 11 Sep 2014 12:00:34 -0400
Subject: [infiniband] mlx4: Fix some indenting in mlx4_ib_add()

Message-id: <674c20e4a18e4dd954f167c67a965690f67d871c.1410425016.git.avadai@redhat.com>
Patchwork-id: 93642
O-Subject: [Patch RHEL7.1 V1 098/197] mlx4_core: Fix some indenting in mlx4_ib_add()
Bugzilla: 1107617 1107618 1107619 1107620
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Dan Carpenter <dan.carpenter@oracle.com>

The code was indented too far and also kernel style says we should have
curly braces.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 3839d8ac1bac55922f6632e36040fdc0de374bba)
Signed-off-by: Amir Vadai <avadai@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/mlx4/main.c b/drivers/infiniband/hw/mlx4/main.c
index 5e0013a..59fd96c 100644
--- a/drivers/infiniband/hw/mlx4/main.c
+++ b/drivers/infiniband/hw/mlx4/main.c
@@ -2048,8 +2048,9 @@ static void *mlx4_ib_add(struct mlx4_dev *dev)
    err = mlx4_counter_alloc(ibdev->dev, &ibdev->counters[i]);
    if (err)
     ibdev->counters[i] = -1;
-  } else
-    ibdev->counters[i] = -1;
+  } else {
+   ibdev->counters[i] = -1;
+  }
  }
 
  mlx4_foreach_port(i, dev, MLX4_PORT_TYPE_IB)
-- 
1.7.1